rest-api/src/main/resources/definitions/participant_file.yml

```diff
@@ -19,4 +19,10 @@ properties:
         type: string
         readOnly: true
         description: "ParticipantFile"
-        x-nullable: false
\ No newline at end of file
+        x-nullable: false
+    expires:
+        type: string
+        format: date-time
+        readOnly: true
+        description: "The time when access to this session expires."
```

Comment:
It would probably be more clear to say "when the downloadUrl or uploadUrl expires".